EIP-2228: Promote to Review (#3302)
ethereum/EIPs: EIPS/eip-2228.md

## Motivation

> The Ethereum network with chainid 1 is referenced using several conflicting names across EIPs (as published at the time EIP 2228 was written), client implementations, and information published on the internet at large:
> 
> * "main net" at [EIP-155](./eip-155.md), [EIP-170](./eip-170.md)
> * "mainnet" at [EIP-86 [DRAFT]](./eip-86.md), [client documentation](https://geth.ethereum.org/support/FAQ)
> * "Main net" at [EIP-2](./eip-2.md), [EIP-150](./eip-150.md)
>    * \* In EIP-2, it is unclear if the first letter is capital because it is a proper noun or if it is the the first word in a text column presented in sentence case
> * "Mainnet" at [EIP-7](./eip-7.md), [EIP-779](./eip-779.md)
> 
> In several examples above, even papers written by the same author use (as published at the time EIP 2228 was written) inconsistent names to refer to the Ethereum network with chainid 1.

Reviewer: ```suggestion
The Ethereum network with chainid 1 is referenced using several conflicting names across EIPs, client implementations, and information published on the internet at large.  In several locations, even documents written by the same author use inconsistent names to refer to the Ethereum network with chainid 1.  Names in use at the time of this writing include:
* `main net`
* `mainnet`
* `Main net`
* `Mainnet`
```

At the least, you'll need to remove draft EIPs from the list and the external link to `https://geth.ethereum.org/support/FAQ`.  I don't believe that linking to these documents provides any future value to this EIP so I recommend just getting rid of all of the links and instead just providing a list of options you have seen used around the internet.

Also, I recommend adding something to the motivation section that explains *why* there is value in consistency here, as I think that is more important than showing that there is a historic lack of consistency.

Author: This is adopted substantially at 199121d174f28f35264980b19cfc96ec8fd8e8fd

## Specification

The network name shall be Ethereum Mainnet, or just Mainnet if the context is known to be discussing Ethereum networks. This IS a proper noun. Several examples are given below which differentiate between usage of the name of the network versus a descriptive reference to the network.

Any name or word styling (i.e. capitalization of the letters) of the network which is inconsistent with the test cases cited below shall NOT be used.

### Trademark note

Note: Ethereum [is a registered trademark](https://tsdr.uspto.gov/#caseNumber=86634529&caseType=SERIAL_NO&searchType=statusSearch) of Ethereum Foundation and [the Foundation asserts](https://ethereum.org/en/terms-of-use/) "you must not use [this mark] without the prior written permission of the Foundation".

> Your application MAY reference "Ethereum&reg; Mainnet" and inculde a note "Ethereum is a registered trademark of Ethereum Foundation" elsewhere.

Reviewer: ```suggestion
Your application MAY reference "Ethereum&reg; Mainnet" and include a note "Ethereum is a registered trademark of Ethereum Foundation" elsewhere.
```

> ### Trademark note
> 
> Note: Ethereum [is a registered trademark](https://tsdr.uspto.gov/#caseNumber=86634529&caseType=SERIAL_NO&searchType=statusSearch) of Ethereum Foundation and [the Foundation asserts](https://ethereum.org/en/terms-of-use/) "you must not use [this mark] without the prior written permission of the Foundation".
> 
> Your application MAY reference "Ethereum&reg; Mainnet" and include a note "Ethereum is a registered trademark of Ethereum Foundation" elsewhere.
> 
> Trademark status, fair use of trademarks, and any written permissions you have with the Ethereum Foundation are outside the scope of this EIP.
> 

Reviewer: The general EIP policy is to not include any external links.  Also, I feel like an EIP is not the appropriate place to mention trademarks or other legal statuses.  If people want to read the ToU they can read the ToU, no need to restate it here.  Your last line of this sums up my position pretty cleanly, but feels unnecessary to include in the EIP.
```suggestion
```

> ### Trademark note
> 
> Note: Ethereum [is a registered trademark](https://tsdr.uspto.gov/#caseNumber=86634529&caseType=SERIAL_NO&searchType=statusSearch) of Ethereum Foundation and [the Foundation asserts](https://ethereum.org/en/terms-of-use/) "you must not use [this mark] without the prior written permission of the Foundation".
> 
> Your application MAY reference "Ethereum&reg; Mainnet" and include a note "Ethereum is a registered trademark of Ethereum Foundation" elsewhere.
> 
> Trademark status, fair use of trademarks, and any written permissions you have with the Ethereum Foundation are outside the scope of this EIP.
> 

Reviewer: The general EIP policy is to not include any external links.  Also, I feel like an EIP is not the appropriate place to mention trademarks or other legal statuses.  If people want to read the ToU they can read the ToU, no need to restate it here.  Your last line of this sums up my position pretty cleanly, but feels unnecessary to include in the EIP.
```suggestion
```

Author: The normative part of this comment is addressed in b5ccfed34f56831fa7d9a1a1becbb87a24a2739a. I refer further discussion to the discussions-to page.
